examples/workspace/BUILD.bazel

```diff
@@ -6,5 +6,29 @@ npm_link_all_packages(name = "node_modules")
 playwright_bin.playwright_test(
     name = "test",
     args = ["test"],
-    data = ["@playwright//:webkit"],
+    data = [
+        "playwright.config.ts",
+        "simple.e2e.ts",
+        "//:node_modules/@playwright/test",
+        "@playwright//:chromium-headless-shell",
+    ],
+    tags = ["requires-network"],
```

Comment:
I'll update the tests myself, but for future PRs network is frowned upon :)

Reply:
Yes, agreed. I wasn't sure how else to make a very simple test